### PR TITLE
Created Anirudh.json

### LINKED ISSUE #173: Add qwe123pro  to the list
I am Anirudh Anup, I would like to add my profile to linkfree

### PR DESCRIPTION
**Pull request by qwe123coder**
Fixes #173 

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/202"></a>

### CODE REVIEW

**Review by FayasNoushad — APPROVED** [review]

**Review by rohan-kulkarni-25 — APPROVED** [review]

**Review by MukulKolpe — APPROVED** [review]

**Comment by schmelto** [comment]
Is polywork supported yet?

**Comment by qwe123coder** [comment]
> Is poly work supported yet?

it is not and Eddie told it will get filtered out and not showed (ig that is what he meant in my other or )

File: public/data/Anirudh.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anirudh Anup",
+  "bio": "I am Anirudh Anup, A passionate 13 year old developer from India, I am always open to collaborating on projects",
+  "avatar": "http://github.com/qwe123coder.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/qwe123coder",
```
Comment (rohan-kulkarni-25) [inline public/data/Anirudh.json:8]: Please add `https` for Github URL

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/data/Anirudh.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anirudh Anup",
+  "bio": "I am Anirudh Anup, A passionate 13 year old developer from India, I am always open to collaborating on projects",
```
Comment (Panquesito7) [inline public/data/Anirudh.json:3]:
```suggestion
  "bio": "A passionate 13-year-old developer from India. I am always open to collaborating on projects",
```

File: public/data/Anirudh.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anirudh Anup",
+  "bio": "A passionate 13-year-old developer from India. I am always open to collaborating on projects",
+  "avatar": "http://github.com/qwe123coder.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/qwe123coder",
```
Comment (Panquesito7) [inline public/data/Anirudh.json:8]:
```suggestion
      "url": "https://github.com/qwe123coder",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/Anirudh.json (hunk 3 above)
Comment (eddiejaoude) [reply public/data/Anirudh.json:8]: GitHub will redirect to https

**Comment by eddiejaoude** [comment]
Any links not supported will not be shown, but later when support will be shown.

I merged the Polywork PR that allows people to add milestones/achievements like in Polywork

**Pull request merged**